councilmatic_core/views.py

```diff
@@ -134,7 +135,10 @@ class CouncilMembersView(ListView):
     context_object_name = 'posts'
     
     def get_queryset(self):
-        return Organization.objects.get(ocd_id=settings.OCD_CITY_COUNCIL_ID).posts.all()
+        try:
+            return Organization.objects.get(ocd_id=settings.OCD_CITY_COUNCIL_ID).posts.all()
+        except ObjectDoesNotExist:
+            return dict()
         # return Post.objects.filter(organization__ocd_id=settings.OCD_CITY_COUNCIL_ID)
```

Comment:
Wait. Can we just bring this back, but with an `OCD_DIVISION_ID`?

We don't currectly have that in `settings_jurisdiction.py`, but it's easily derivable from `OCD_JURISDICTION_ID`:

```py
OCD_JURISDICTION_ID = 'ocd-jurisdiction/country:us/state:il/place:chicago/government'
OCD_DIVISION_ID = '/'.join(OCD_JURISDICTION_ID.split('/')[:-1])
```